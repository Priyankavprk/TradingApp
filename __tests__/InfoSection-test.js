import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import InfoSection from '../components/infoSection';

configure({ adapter: new Adapter() });

jest.mock('@fortawesome/react-native-fontawesome', () => ({
  FontAwesomeIcon: ''
}));

it('flatlist return keyExtractor correctly', () => {
  const wrapper = shallow((<InfoSection />));
  const key = wrapper.find('FlatList').props().keyExtractor('open', 0);
  expect(key).toEqual('open0')
});