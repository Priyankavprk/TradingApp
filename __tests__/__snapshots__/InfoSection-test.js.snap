// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`flatlist renderItem correctly 1`] = `ShallowWrapper {}`;
